.github/workflows/ci.yml

```diff
@@ -0,0 +1,67 @@
+name: Test and lint
+
+on:
+  push:
+  pull_request:
+
+permissions:
+  contents: read
+
+jobs:
+  tests:
+    name: Run tests
```

Comment:
```suggestion
    name: ${{ matrix.python-version }} tests
```

Reply:
The matrix parameters are included in the name automatically by GitHub, so I don't think this is useful.